js-exercises/sum-all/sumAll.js

```diff
@@ -0,0 +1,44 @@
+const isNumber = (arg) => typeof arg === 'number';
+
+const areNumbers = (args) => args.every(isNumber);
```

Comment:
This would be true for `NaN` as well.

Reply:
right, issue fixed in 1e9a59a7b31f1961f18cfa90dab8749e73646be6